Controller/CRUDController.php

```diff
@@ -456,6 +473,8 @@ public function editAction($id = null, Request $request = null)
                 $templateKey = 'preview';
                 $this->admin->getShow();
             }
+        } else {
+            $session->set($versionKey, $this->admin->getVersion($object));
```

Comment:
@rande If I move the locking logic in an extension, how can I achieve this ? I need to do this only when the object is loaded for the first time and the form is not submited yet.